lib/instana/tracer.rb

```diff
@@ -180,11 +180,7 @@ def log_exit(name, kvs = {})
       self.current_span.set_tags(kvs)
       self.current_span.close
 
-      if self.current_span.parent
-        self.current_span = self.current_span.parent
-      else
-        self.current_span = nil
-      end
```

Comment:
Apparently there is a new rubocop version and this line triggered

```
lib/instana/tracer.rb:183:7: C: [Correctable] Style/RedundantCondition: Use double pipes || instead.
      if self.current_span.parent ...
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

This fix was created by `rubocop -a`.